test/es6/ES6RestrictedProperties.js

```diff
@@ -369,7 +364,7 @@ var tests = [
         body: function () {
             var obj = { func() { } }

```

Comment:
>var obj = { func() { } } [](start = 12, length = 24)

Do we need to have cases like

var obj = { func:function() { } }
function foo() {}
var obj = { func:foo }
var obj = { func:Math.sin }

Reply:
These cases are not methods - they should behave like ordinary functions.

Reply:
Good to add test to verify that?

---
In reply to: [174215572](https://github.com/Microsoft/ChakraCore/pull/4810#discussion_r174215572) [](ancestors = 174215572)